server/views/pages/home.njk

```diff
@@ -11,62 +11,72 @@
     {% set isFemalePrison = true %}
 {% endif %}
 
-{% block content %}
+{% block skipLink %}
```

Comment:
added `skipLink` block to override `main` block behaviour and ensure that the skip link takes you to the homepage content